keyboards/handwired/stream_cheap/2x5/keymaps/default/keymap.c

```diff
@@ -0,0 +1,57 @@
+#include QMK_KEYBOARD_H
+
+
+enum custom_keycode {
+    VS_COMMENT = SAFE_RANGE,
+    VS_UNCOMMENT,
+    TST_STRING
+};
+
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case VS_COMMENT:
+            if (record->event.pressed) {
+                SEND_STRING(SS_LCTL("kc"));
+            }
+            break;
+        case VS_UNCOMMENT:
+            if (record->event.pressed) {
+                SEND_STRING(SS_LCTL("ku"));
+            }
+            break;
+        case TST_STRING:
+            if (record->event.pressed) {
+                SEND_STRING("12345"SS_TAP(X_ENTER));
```

Comment:
```suggestion
                SEND_STRING("12345\n");
```

Reply:
Just curious what is different between /n and ss_tap(x_enter) ?

I'll make the change I was just curious

Reply:
Functionally, there is no difference, the enter key will still be sent. However `SS_TAP(X_ENTER)` compiles to `0x01 0x01 0x28` ([The first two bytes represent a tap event](https://github.com/qmk/qmk_firmware/blob/master/quantum/send_string_keycodes.h#L394)), whereas simply adding `\n` to the string results in just `0x0A`, the ASCII line feed character, which is already mapped to `KC_ENTER` in the [sendstring lookup table](https://github.com/qmk/qmk_firmware/blob/master/quantum/quantum.c#L391). So, it just saves a couple bytes and a bit of extra computation.